samples/nrf9160/spm/nrf9160dk_nrf9160.overlay

```diff
@@ -1,8 +1,4 @@
-/* Needed to get NRF_UARTE2 and NRF_PWMn defined. */
```

Comment:
cc @jtguggedal

Reply:
@lemrey I think this was originally put here in order to build the BLE LTE gateway. Are you okay with removing it?